pkg/liqonet/iptables/iptables.go

```diff
@@ -286,7 +335,7 @@ func (h IPTHandler) EnsureChainRulesPerCluster(tep *netv1alpha1.TunnelEndpoint)
 		if err != nil {
 			return fmt.Errorf("cannot get existing chain rules per cluster %s: %w", tep.Spec.ClusterIdentity, err)
 		}
-		if err := h.updateSpecificRulesPerChain(chain, rules, newRules); err != nil {
+		if err := h.updateSpecificRulesPerChain(chain, rules, newRules, false, Append); err != nil {
```

Comment:
Can we define a const for this "false"? Because it is quite hard to read and understand in this way